docs/element-definitions.md

```diff
@@ -1353,6 +1356,168 @@ the `inferred` link tells the user that the element was created based on some
 varying assumptions, and the URL to which the link points MAY provide an
 explanation on how and why it was inferred.
 
+## Authentication
+
+Example:
+
+```json
+{
+  "element": "category",
+  "meta": {
+    "classes": ["api"]
+  },
+  "content": [
+    {
+      "element": "category",
+      "meta": {
+        "classes": ["authenticationSchemes"]
+      },
+      "content": [
+        {
+          "element": "auth:basic",
+          "meta": {
+            "id": "Custom Basic Auth 1"
+          },
+          "attributes": {
+            "username": "john",
+            "password": {
+              "element": "string",
+              "content": "p455w0rd"
+            }
+          }
+        },
+        {
+          "element": "auth:basic",
+          "meta": {
+            "id": "Custom Basic Auth 2",
+            "description": "Username/password not required?"
+          }
+        },
+        {
+          "element": "auth:oath2",
+          "meta": {
+            "id": "My Oauth2"
+          },
+          "attributes": {
+            "scopes": [
+              {
+                "element": "auth:oauth2Scope",
```

Comment:
I don't think we need this element. We can have a `member` `scopes` in `content` which is an array.